<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_shared</class>
 <widget class="QWidget" name="Page_shared">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="shared_file_tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>621</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <column>
    <property name="text">
     <string>选择</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>大小</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>修改日期</string>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
     </font>
    </property>
    <property name="background">
     <color>
      <red>218</red>
      <green>218</green>
      <blue>218</blue>
     </color>
    </property>
   </column>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="bt_share_file">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>生成分享链接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_delete_share">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_add_share_file">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="text">
    <string> 共享文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
